<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_export_results</class>
 <widget class="QDialog" name="Dialog_export_results">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export results</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox_general_settings">
     <property name="title">
      <string>General settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_select_lsm_run">
          <property name="text">
           <string>Select LSM run</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBox_select_lsm_run">
          <property name="toolTip">
           <string>Select one LSM run for data export.</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_export_path">
          <property name="text">
           <string>Export_path</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_export_path">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Default: General campaign path.</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_ndb_file">
     <property name="title">
      <string>nsb file</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="checkBox_write_nsb_file">
        <property name="text">
         <string>Write nsb file</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="toolTip">
         <string>Take vertical offsets between instrument top and ground (dhb) and instrument top and reference point (dhf) at a station from the first setup at this station.</string>
        </property>
        <property name="text">
         <string>Take dhb and dhf from first setup at a station</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_dhb_dhf</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="toolTip">
         <string>Calculate mean values of the vertical offsets between instrument top and ground (dhb) and instrument top and reference point (dhf) from all setups at a station. Write these mean values to the nsb file.</string>
        </property>
        <property name="text">
         <string>Use mean dhb and dhf at stations</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_dhb_dhf</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_other_files">
     <property name="title">
      <string>Other files</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="checkBox_2">
        <property name="text">
         <string>write log file (text file)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>create drift plot (PDF)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_export_results</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_export_results</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup_dhb_dhf"/>
 </buttongroups>
</ui>
